financial-templates-lib/test/ExpiringMultiPartyClient.js

```diff
@@ -29,11 +30,10 @@ contract("ExpiringMultiPartyClient.js", function(accounts) {
     await collateralToken.mint(accounts[0], toWei("100000"), { from: accounts[0] });
     await collateralToken.mint(accounts[1], toWei("100000"), { from: accounts[0] });
 
-    // Create identifier whitelist and register the price tracking ticker with it.
-    identifierWhitelist = await IdentifierWhitelist.new();
+    identifierWhitelist = await IdentifierWhitelist.deployed();
```

Comment:
This is admittedly outside the scope of the PR, but there was redundant code in adding price identifiers to the whitelist.